buildlib/build/string.d

```diff
@@ -0,0 +1,25 @@
+module build.string;
+
+auto firstLine(T)(T str)
+{
+    import std.string;
+
+    auto newlineIndex = str.indexOf("\n");
+    if (newlineIndex < 0)
+        return str;
```

Comment:
Why not `stripLeft`?

Reply:
Ah I was referring to stripLeft's second argument (you can do, e.g. `str.stripLeft("\n"))`), but in this case it looks like you want `until`.